build/Containerfile.tools

```diff
@@ -1,8 +1,10 @@
+# hadolint ignore=DL3007
```

Comment:
Same here, I do not know what `DL3007` is, why it is ignored, or why it can not be fixed.

Reply:
It is for "Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag". 
So, should I put the latest release version here or keep it ignored?